File: online-judge-solutions/Codeforces/1722B.py
#https://codeforces.com/problemset/problem/1722/B
#Lang		:	Python 3.8
#Time		:	31 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n,a,b=int(input()),input(),input()
    a,b=a.replace('G','B'),b.replace('G','B')
    print("YES" if a==b else "NO")

File: online-judge-solutions/Codeforces/1722B.cpp
//https://codeforces.com/problemset/problem/1722/B
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
#include<algorithm>
using namespace std;
int main(){
	int t,n;
	cin>>t;
	while(t--){
		string a,b;
		cin>>n>>a>>b;
		replace(begin(a),end(a),'G','B');
		replace(begin(b),end(b),'G','B');
		cout<<(a==b?"YES":"NO")<<"\n";
	}
	return 0;
}
